client/src/pages/UserProfile.js

```diff
@@ -1,53 +1,278 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { makeStyles } from '@material-ui/core/styles';
-import Card from '@material-ui/core/Card';
-import CardActions from '@material-ui/core/CardActions';
-import CardContent from '@material-ui/core/CardContent';
-import Button from '@material-ui/core/Button';
+import { Grid, TextField, Card, FormControl, 
+          InputLabel, Select, MenuItem, Button, CardContent } from '@material-ui/core'
 import Typography from '@material-ui/core/Typography';
-
+import { KeyboardTimePicker, KeyboardDatePicker } from '@material-ui/pickers';
+ 
 const useStyles = makeStyles({
   root: {
+    flexGrow: 1,
     minWidth: 275,
-  },
-  bullet: {
-    display: 'inline-block',
-    margin: '0 2px',
-    transform: 'scale(0.8)',
+    margin: '4px 4px',
+    padding: '10px 10px',
+    direction: "column",
+    alignItems: "center",
+    justify: "center"
   },
   title: {
-    fontSize: 14,
+    fontSize: 22,
+    fontWeight: "bold",
+    textAlign: 'center'
+  },
+  formControl: {
+    margin: '4px 4px',
+    minWidth: 120,
   },
-  pos: {
-    marginBottom: 12,
+  selectEmpty: {
+    marginTop: '4%',
   },
 });
 
 export default function UserProfile() {
   const classes = useStyles();
-  const bull = <span className={classes.bullet}>•</span>;
+  const [firstName, setFirstName] = useState('')
+  const [lastName, setLastName] = useState('')
+  const [gender, setGender] = useState('')
+  const [selectedDate, setSelectedDate] = useState(new Date('2000-01-01T21:11:54'))
+  const [emailAddress, setEmailAddress] = useState('')
+  const [phoneNumber, setPhoneNumber] = useState('')
+  const [addressOne, setAddressOne] = useState('')
+  const [addressTwo, setAddressTwo] = useState('')
+  const [city, setCity] = useState('')
+  const [province, setProvince] = useState('')
+  const [zipCode, setZipCode] = useState('')
+  const [country, setCountry] = useState('')
+  const [aboutme, setAboutMe] = useState('Tell Us about yourself')
+  
```

Comment:
maybe use one useState for profile object with all fields together?

Reply:
great point - i'm personally a fan of this